github_connector/demo/github_organization.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright (C) 2016-Today: Odoo Community Association (OCA)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+    <record id="oca_organization" model="github.organization">
```

Comment:
I think this OCA demo as repo is not a good idea: given the size of the repo if somebody triggers a synchronization, it will be either too long or it will break travis.
I dont know if it is possible to use a small repo instead or create a demo repo (oca_demo)

Reply:
I don't know too. Can you create a oca demo repo?

Reply:
Travis is not affected here (maybe Runbot), but there's no other organization to be used here, except a private one. As you can trigger the fetch of repositories, branches and so on in a separate way, I think it's tolerable. What we can maybe do is to increase the list of the blacklist for limiting the number of the repositories, but this is something weird IMO.

Reply:
I meant runbot indeed.
Currently you cannot test runbot because you cannot pull any repo (OCA would break runbot or take forever).
This is why I suggest to create a demo repository in github (a public one, owned by OCA_demo entity) with one module inside